### DISCUSSION TITLE
Avoid deletion of topics

### DISCUSSION THREAD

**Discussion by shreyasarani23** [strimzi/strimzi-kafka-operator#4975] [General]
Does Strimzi provide any way to control the deletion of topic. Suppose say that I am kafka cluster administrator and multiple users are working on it and they have access to the yaml files. Is there a way to configure role based access at the topic level such that only cluster administrator has the privilege to delete,update,create topics and rest other should not have access.

**Comment by scholzj** [comment]
On Kafka level, you can use authorization and Kafka ACLs to decide who can or cannot delete which topic. On Kubernetes level, you can use Kubernetes RBAC to decide who can use which resources - you can protect the KafkaTopic resource in the same way as any other Kubernetes resource.

**Reply by shreyasarani23** [reply]
can u share me the link where they have used authorization and kafka ACL's for the kafka topics?

**Reply by scholzj** [reply]
Are you sure you cannot find it in the documentation?

**Reply by shreyasarani23** [reply]
I can find authorization enabled for kafka broker,kafka user, kafka connect etc. Is the same applicable for kafka topics as well ? or w.r.t to topics is it different?

**Reply by scholzj** [reply]
The Kafka authorization applies to every user who connects to Kafka using the Kafka protocol.

**Reply by shreyasarani23** [reply]
Ok got it. Thanks @scholzj
